RecoBTag/SecondaryVertex/plugins/TemplatedSecondaryVertexProducer.cc

```diff
@@ -273,15 +278,16 @@ TemplatedSecondaryVertexProducer<IPTI,VTX>::TemplatedSecondaryVertexProducer(
 	    constraint == CONSTRAINT_BEAMSPOT ||
 	    constraint == CONSTRAINT_PV_PRIMARIES_IN_FIT )
 	    token_BeamSpot = consumes<reco::BeamSpot>(params.getParameter<edm::InputTag>("beamSpotTag"));
-        useExternalSV = false;
-        if(params.existsAs<bool>("useExternalSV")) useExternalSV = params.getParameter<bool> ("useExternalSV");
-        if(useExternalSV) {
+	useExternalSV = false;
+	if(params.existsAs<bool>("useExternalSV")) useExternalSV = params.getParameter<bool> ("useExternalSV");
```

Comment:
I think you are right but the rule is likely being enforced for newly added modules. In this case no new modules or parameters were added. Note that there are plenty of cases like this one across the b-tagging code. It would be quite an effort to update all that code.